services/mailgun.js

```diff
@@ -0,0 +1,42 @@
+const Mailgun = require('mailgun-js');
+require('dotenv').config()
+
+
+//Your api key, from Mailgun’s Control Panel
+const api_key = process.env.API_KEY
```

Comment:
good variable names should make adding a comment unnecesssary. If you used `MAILGUN_API_KEY`, then you don't need the comment.